atlp-rwanda/dynamites-ecomm-be
# Implement get client api address and location

## Description
This pull request adds a new column named `country` to the `order_details` table in our database. This column will store the country information for each order, which is currently missing from our data model. Adding this column will help us track the geographical distribution of our sales and improve our data analytics.

## Acceptance Criteria
- [x] - A new column named `country` is added to the `order_details` table.
- [x] - The column can store country names in string format.
- [x] - The column should allow NULL values initially to accommodate existing records without country information.
- [x] - Write tests to verify that the new column is correctly added.

## How should this be manually tested?
- Checkout this branch `git checkout fix-get-couuntry-#126`
- Install all dependencies `npm install`
- Start the server `npm run dev`
- Ensure endpoints "localhost:3000/api/v1/category/getSalesByCountry"  it should return list of country;

## Additional Notes
- Ensure that the new column does not negatively impact the performance of the `order_details` table.
